src/Symfony/Component/DependencyInjection/Exception/ParameterNotFoundException.php

```diff
@@ -103,4 +108,16 @@ public function setSourceExtensionName(?string $sourceExtensionName): void
 
         $this->updateRepr();
     }
+
+    public function getExtraMessage(): ?string
+    {
+        return $this->extraMessage;
+    }
+
+    public function setExtraMessage(?string $extraMessage): void
```

Comment:
do we really need the getter and setter?

Reply:
I think they are if someone wants to add an extra message by catching and rethrowing